usaspending_api/awards/v2/filters/sub_award.py

```diff
@@ -207,7 +273,7 @@ def recip_string_parse(recipient_string):
                 raise InvalidParameterException("Invalid filter: recipient_scope type is invalid.")
 
         elif key == "recipient_locations":
-            queryset = queryset.filter(geocode_filter_locations("recipient_location", value))
+            queryset = queryset.filter(geocode_filter_subaward_locations("sub_legal_entity", value))
```

Comment:
This swap or find and replace may not be correct. For `recipient_locations` ... should it be referencing `"sub_legal_entity"`?

Reply:
It should actually be referencing `sub_legal_entity`:

Subaward -> SubawardSearch
recipient_location_x -> sub_legal_entity_x
(ex. recipient_location_state_code -> sub_legal_entity_state_code)